llvm/test/CodeGen/AMDGPU/reduce-frame-index.mir

```diff
@@ -0,0 +1,176 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 5
+# RUN: llc -mtriple=amdgcn-amd-amdhsa -mcpu=gfx90a -verify-machineinstrs -run-pass=prologepilog %s -o - | FileCheck %s
+--- |
```

Comment:
Reduce the test further?

Reply:
after reducing further bug is not producible.

Reply:
Test name should be better, it's too generic

Reply:
thanks for the suggestion, i have renamed it.